src/services/getCitiesData.js

```diff
@@ -4,11 +4,7 @@ import getData from './getData';
 export default async function getCitiesData(query) {
   try {
     const { locations } = await getData(`${locationURL}${query}`);
```

Comment:
It is still fine to have a layer, kinda DTO where you prepare the data after fetch. But just make sure that you are not filtering or corrupting useful data on this stage.